### PR TITLE
uORB topics: Moved all to auto-generated

### PR DESCRIPTION
**Pull request by LorenzMeier**
@againagainst @DrTon this moves all topics to auto-generation. When the generator now supports to mark a message to be suitable for multiple topics the complete objects_common file can be generated as well.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@kd0aij I had to fiddle quite a bit with the RC to param mapping code. Could you maybe cross-test if your setup still works with this branch?

File: src/drivers/px4fmu/fmu.cpp [hunk 1]
```diff
@@ -607,7 +607,7 @@ PX4FMU::task_main()
 	orb_advert_t to_input_rc = 0;
 
 	memset(&rc_in, 0, sizeof(rc_in));
-	rc_in.input_source = RC_INPUT_SOURCE_PX4FMU_PPM;
+	rc_in.input_source = input_rc_s::RC_INPUT_SOURCE_PX4FMU_PPM;
```
Comment (kd0aij) [inline src/drivers/px4fmu/fmu.cpp]: lines 775 and 776 also need input_rc_s:: added

**Comment by LorenzMeier** [comment]
@kd0aij Does the RC mapping now work or not?

**Comment by kd0aij** [comment]
RC parameter setting looks good

**Comment by LorenzMeier** [comment]
Thanks!

**Pull request merged**